### PR TITLE
Added missing command to Config guide.

### LINKED ISSUE #1812: Simple deployment method does not work in 2.2.3 and there's a typo in the code
Feedback on page: /guides/v2.2/config-guide/config/disable-module-output.html

### PR DESCRIPTION
**Pull request by dobooth**
## This PR is a:

- Content fix or rewrite

## Summary

When this pull request is merged, it will...

Updates /guides/v2.2/config-guide/config/disable-module-output.html to include a missing command.
Closes #1812 

whatsnew
Added the missing `cache` command in [Disable Module Output](https://devdocs.magento.com/guides/v2.2/config-guide/config/disable-module-output.html#export-the-configuration-changes).

### CODE REVIEW

File: guides/v2.2/config-guide/config/disable-module-output.md [hunk 1]
```diff
@@ -7,31 +7,33 @@ functional_areas:
...
+For example, the `Customer` module depends on the `Review` module, so the `Review` module can not be disabled.
+However, if you do not want customers to be able to provide reviews, you could turn off output from the `Review` module.
 
 {:.bs-callout .bs-callout-info}
 If a merchant used the Admin to disable a module's output in a previous release, you must manually configure the system to migrate these settings.
 
 ## Disable module output in a pipeline deployment
 
-To disable module output in pipeline or any other deployment with multiple instances of Magento:.
+To disable module output in pipeline or any other deployment with multiple instances of Magento:
```
Comment (shrielenee) [inline guides/v2.2/config-guide/config/disable-module-output.md]:
```suggestion
To disable module output in the pipeline, or any other deployment, with multiple instances of Magento:
```

File: guides/v2.2/config-guide/config/disable-module-output.md [hunk 2]
```diff
@@ -7,31 +7,33 @@ functional_areas:
...
 
 1. Edit the `Backend` module's `config.xml` file.
```
Comment (shrielenee) [inline guides/v2.2/config-guide/config/disable-module-output.md]: Is Backend capitalized in the code?

File: guides/v2.2/config-guide/config/disable-module-output.md [hunk 3]
```diff
@@ -7,31 +7,33 @@ functional_areas:
...
-2. Export the configuration changes.
+1. Export the configuration changes.
 
 ### Edit the `Backend` module's `config.xml` file
 
-Archive the original `config.xml` file. Then add lines similar to the following to the `<Magento_install_dir>/vendor/magento/module-backend/etc/config.xml` file, directly under the `<default>` element:
+Archive the original `config.xml` file. 
+Then add lines similar to the following to the `<Magento_install_dir>/vendor/magento/module-backend/etc/config.xml` file, directly under the `<default>` element:
```
Comment (shrielenee) [inline guides/v2.2/config-guide/config/disable-module-output.md]: If you're adding transitional words like "then", consider converting to numbered steps (if there are 2 or more steps).

File: guides/v2.2/config-guide/config/disable-module-output.md [hunk 4]
```diff
@@ -45,19 +47,28 @@ As a sample result of this configuration, customers can no longer sign up to rec
...
+
 For more information about this command, see [Export the configuration]({{ page.baseurl }}/config-guide/cli/config-cli-subcommands-config-mgmt-export.html).
 
 ## Disable module output in a simple deployment
 
 The procedure for disabling module output on a single instance of Magento is easier because the changes don't have to be distributed.
 
-Archive the original `<Magento_install_dir>/app/etc/config.php` file. Then add the `advanced` and `modules_disable_output` sections to the `config.php` file (if they don't already exist):
+Archive the original `<Magento_install_dir>/app/etc/config.php` file.
+Then add the `advanced` and `modules_disable_output` sections to the `config.php` file (if they don't already exist):
```
Comment (shrielenee) [inline guides/v2.2/config-guide/config/disable-module-output.md]: Numbered steps?

**Comment by shrielenee** [comment]
If this is indeed a technical change, make sure you add a whatsnew to the PR descrip. (Maybe you do that after, I dunno --- just a reminder!)

File: guides/v2.2/config-guide/config/disable-module-output.md (hunk 2 above)
Comment (keharper) [reply guides/v2.2/config-guide/config/disable-module-output.md]: Yes

File: guides/v2.2/config-guide/config/disable-module-output.md (hunk 1 above)
Comment (keharper) [reply guides/v2.2/config-guide/config/disable-module-output.md]: I would say "in a pipeline deployment or in any other deployment..."

File: guides/v2.2/config-guide/config/disable-module-output.md [hunk 5]
```diff
@@ -45,19 +47,28 @@ As a sample result of this configuration, customers can no longer sign up to rec
 
 Run the following command to export the configuration changes:
 
-`magento app:config:dump`
+```bash
+magento app:config:dump
```
Comment (keharper) [inline guides/v2.2/config-guide/config/disable-module-output.md]:
```suggestion
bin/magento app:config:dump
```

File: guides/v2.2/config-guide/config/disable-module-output.md [hunk 6]
```diff
@@ -78,8 +89,8 @@ Archive the original `<Magento_install_dir>/app/etc/config.php` file. Then add t
 
 Here:
 
- - The `array` beneath `modules_disable_output` contains a list of modules. 
- - `Magento_Review` specifies which module to disable output for.
- - `1` is the flag that disables output for the `Magento_Review` module.
+- The `array` beneath `modules_disable_output` contains a list of modules.
+- `Magento_Review` specifies which module to disable output for.
+- `1` is the flag that disables output for the `Magento_Review` module.
```
Comment (keharper) [inline guides/v2.2/config-guide/config/disable-module-output.md]: I would rewrite lines 93-94 to something like 

In this example, output for the `Magento_Review` module has been disabled. To re-enable output, set the value to `0`.

File: guides/v2.2/config-guide/config/disable-module-output.md (hunk 1 above)
Comment (dobooth) [reply guides/v2.2/config-guide/config/disable-module-output.md]: Fixed.

File: guides/v2.2/config-guide/config/disable-module-output.md (hunk 2 above)
Comment (dobooth) [reply guides/v2.2/config-guide/config/disable-module-output.md]: Yes. It is a module name.

File: guides/v2.2/config-guide/config/disable-module-output.md (hunk 3 above)
Comment (dobooth) [reply guides/v2.2/config-guide/config/disable-module-output.md]: Got it.

File: guides/v2.2/config-guide/config/disable-module-output.md (hunk 4 above)
Comment (dobooth) [reply guides/v2.2/config-guide/config/disable-module-output.md]: Yes.

File: guides/v2.2/config-guide/config/disable-module-output.md (hunk 6 above)
Comment (dobooth) [reply guides/v2.2/config-guide/config/disable-module-output.md]: Got it.

File: guides/v2.2/config-guide/config/disable-module-output.md (hunk 5 above)
Comment (dobooth) [reply guides/v2.2/config-guide/config/disable-module-output.md]: Fixed.

**Comment by dobooth** [comment]
running tests

**Pull request merged**